pandas/core/ops.py

```diff
@@ -745,24 +740,19 @@ def na_op(x, y):
         elif is_categorical_dtype(y) and not is_scalar(y):
             return op(y, x)
 
-        if is_object_dtype(x.dtype):
+        elif is_object_dtype(x.dtype):
             result = _comp_method_OBJECT_ARRAY(op, x, y)
+
+        elif is_datetimelike_v_numeric(x, y):
```

Comment:
tests that hit this

Reply:
tests.series.test_operators.TestSeriesComparisons.test_comparison_invalid.  This is just moved up a few lines and down one level of indentation.

Reply:
ok